repo_tests/provision_playbook_path_agrees_15707_test.py

```
@@ -0,0 +1,144 @@
+    text = _strip_comments(_PLAYBOOK.read_text(encoding="utf-8"))
+    match = re.search(r'^\s*provision_playbook:\s*"([^"]+)"\s*$', text, re.MULTILINE)
+    assert match, "provision_playbook is no longer a plain literal in the playbook"
+    return match.group(1)
+
+
+def default_repo_path() -> str:
+    """`DEFAULT_REPO_PATH`'s fallback, read from git_tracker rather than restated.
+
+    Deliberately does **not** strip comments, unlike the readers above, and is
+    safe only because the pattern is anchored `^DEFAULT_REPO_PATH` with no
+    leading `\s*`: a commented-out copy is indented behind `# `, so the anchor
```

Ruff (0.16.3)

[warning] Invalid escape sequence: `\s`

Use a raw string literal

(W605)